pages/ledger/index.vue

```diff
@@ -9,6 +9,33 @@
         @click.native="connect"
       />
     </div>
+    <div v-if="isWindows && !hasHIDEnabled">
+      Due to recent Ledger updates, using a Ledger on Windows now requires
+      "Experimental Web Platform features" to be enabled.
```

Comment:
this is chrome dependant so isChrome should be included in the v-if
there is also a v-else missing explaining that Ledger will not work and the user should use Chrome or Brave

Reply:
oh I see. Thank you